pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -535,7 +556,7 @@ func handleDistributionSeries(series timeSeries, defaultMetricName string, serie
 	})
 }
 
-func (e *CloudMonitoringExecutor) parseResponse(queryRes *tsdb.QueryResult, data cloudMonitoringResponse, query *cloudMonitoringQuery) error {
+func (query *cloudMonitoringQuery) parseResponse(queryRes *tsdb.QueryResult, data cloudMonitoringResponse) error {
```

Comment:
[The name of a method's receiver should be a reflection of its identity; often a one or two letter abbreviation of its type suffices (such as "c" or "cl" for "Client")](https://github.com/golang/go/wiki/CodeReviewComments#receiver-names):
```suggestion
func (q *cloudMonitoringQuery) parseResponse(queryRes *tsdb.QueryResult, data cloudMonitoringResponse) error {
```